test/e2e/performanceprofile/functests/1_performance/netqueues.go

```diff
@@ -55,11 +55,16 @@ var _ = Describe("[ref_id: 40307][pao]Resizing Network Queues", Ordered, Label(s
 
 		tunedPaoProfile := fmt.Sprintf("openshift-node-performance-%s", performanceProfileName)
 		//Verify the tuned profile is created on the worker-cnf nodes:
-		tunedCmd := []string{"tuned-adm", "profile_info", tunedPaoProfile}
+		// direct the error to /dev/null on purpose because tuneD always shows the following error:
+		// "Cannot talk to TuneD daemon via DBus. Is TuneD daemon running?"
+		// Which causes the test to fail, but it's a false-positive
+		tunedCmd := []string{"/bin/sh", "-c", fmt.Sprintf("tuned-adm profile_info %s 2>/dev/null | grep ^openshift-", tunedPaoProfile)}
```

Comment:
I get (and I fully agree) with your intent but arguably redirecting stderr is another form of ignoring errors
But still I like more this variant because the test is stricter and seems more targeted